rali/rali/source/rali_api_augmentation.cpp

```diff
@@ -1766,10 +1767,11 @@ raliSSDRandomCrop(
         output_info.height(input->info().height_single());
         output = context->master_graph->create_image(output_info, is_output);
         output->reset_image_roi();
-        std::shared_ptr<RandomCropNode> crop_node =  context->master_graph->add_node<RandomCropNode>({input}, {output});
+        std::shared_ptr<SSDRandomCropNode> crop_node;
+        crop_node =  context->master_graph->add_node<SSDRandomCropNode>({input}, {output});
```

Comment:
Codacy found an issue: [Either the condition '!p_context' is redundant or there is possible null pointer dereference: context.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6302037)